### ISSUE TITLE
BUG: UI-Pod not starting with SecurityContext set

### ISSUE THREAD

**Issue by christianhuth** [kubevious/helm#13]
When using SecurityContext and PodSecurityContext to run the ui-pod as nonRoot, i receive the following error in the pod logs:

```
sed: can't create temp file '/etc/nginx/conf.d/default.confXXXXXX': Permission denied
sed: can't create temp file '/etc/nginx/conf.d/default.confXXXXXX': Permission denied
2021/03/30 14:36:27 [warn] 9#9: the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /etc/nginx/nginx.conf:2
nginx: [warn] the "user" directive makes sense only if the master process runs with super-user privileges, ignored in /etc/nginx/nginx.conf:2
2021/03/30 14:36:27 [emerg] 9#9: host not found in resolver "$DNS_SERVERS" in /etc/nginx/conf.d/default.conf:12
nginx: [emerg] host not found in resolver "$DNS_SERVERS" in /etc/nginx/conf.d/default.conf:12
```

The easiest solution to make this run in a cluster with PodSecurityPolicies activated woule be to be able to set the serviceaccount under which the pod should be started and give this serviceaccount the privilege to run as root.

**Comment by rubenhak** [comment]
@christianknell, could you include complete configuration (helm overrides and other custom yaml you applied). 

As a solution are you recommending to add an ability to set `spec.template.spec.serviceAccountName` on ui pod?

Thanks.

**Comment by rubenhak** [comment]
Included in version 0.9.10
